frontend/javascripts/oxalis/view/action_bar_view.tsx

```diff
@@ -250,11 +250,10 @@ class ActionBarView extends React.PureComponent<Props, State> {
     const isViewMode = controlMode === ControlModeEnum.VIEW;
     const isArbitrarySupported = hasSkeleton || isViewMode;
     const isAIAnalysisEnabled = () => {
-      const activeUser = this.props.activeUser;
-      const jobsEnabled = features().jobsEnabled;
-      if (isViewMode) {
-        return jobsEnabled && activeUser != null && activeUser.isSuperUser;
-      }
+      const jobsEnabled =
+        features().jobsEnabled &&
+        (dataset.dataStore.jobsSupportedByAvailableWorkers.includes(APIJobType.INFER_NEURONS) ||
+          dataset.dataStore.jobsSupportedByAvailableWorkers.includes(APIJobType.INFER_NUCLEI));
```

Comment:
It looks like isAIAnalysisEnabled is only used to call renderStartAIJobButton which then uses `setAIJobModalStateAction("neuron_inferral")`. So, no nuclei? Maybe I'm missing something.

Reply:
Yes, the UI only let's you trigger neuron jobs ATM. Nucleii maybe reactivated in the future, but the job types enum still defines them so I included them for completeness. Hence an `OR` condition.